backend/src/main/kotlin/org/loculus/backend/api/SubmissionTypes.kt

```diff
@@ -344,3 +361,25 @@ class CompressionFormatConverter : Converter<String, CompressionFormat> {
     }
         ?: throw IllegalArgumentException("Unknown compression: $source")
 }
+
+typealias SubmissionIdFilesMap = Map<SubmissionId, FileColumnNameMap>
+
+fun SubmissionIdFilesMap.getAllFileIds(): Set<FileId> = this.values.flatMap {
+    it.values
+}.flatten().map { it.fileId }.toSet()
+
+/**
+ * A map from fileFieldIds to lists of file IDs and names.
```

Comment:
What are `fileFieldIds`? I can't find the term used anywhere else in the PR, is this the "type" of file like "raw_reads"?